.env.example

```diff
@@ -20,7 +20,5 @@ API_KEYS=SCOROTEST,ROBOTO,SCORO
 API_KEY_SCORO_TEST=SCOROTEST
 API_KEY_ROBOT=ROBOTO
 API_KEY_SCORO=SCORO
-MEILI_HTTP_ADDR="http://meilisearch:7700"
-MEILI_MASTER_KEY="ghvjkgisbgkbgskegblfqbgjkebbhgwkjfb"
```

Comment:
Pretty sure you need the master key, meilisearch does not run in prod mode without.

Reply:
We can add it for prod, not needed in the ci I think